src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -396,6 +396,8 @@ private function getStubsDirectory(): string
             }
         }
 
+        // @codeCoverageIgnoreStart
```

Comment:
Should drop this :thinking:

Reply:
Alternatively: document why it's not covered (impossible scenario, I suppose?)

Reply:
Yes, impossible scenario - not possible to change value of private constant.